clar2wasm/src/datastore.rs

```diff
@@ -566,11 +567,11 @@ impl BurnStateDB for BurnDatastore {
     /// the epoch enclosing `height`.
     fn get_stacks_epoch(&self, _height: u32) -> Option<StacksEpoch> {
         Some(StacksEpoch {
-            epoch_id: StacksEpochId::latest(),
+            epoch_id: StacksEpochId::Epoch25,
             start_height: 0,
             end_height: u64::MAX,
             block_limit: ExecutionCost::max_value(),
-            network_epoch: PEER_VERSION_EPOCH_2_1,
+            network_epoch: PEER_VERSION_EPOCH_2_5,
```

Comment:
Was badly exported in the previous version of clarity. `PEER_VERSION_EPOCH_2_5`

This datastore is only used in tests, I don't _really_ car about the actual value here I believe, at least for the current tests suite. We might want a finer control of the epoch here in the future

Reply:
> We might want a finer control of the epoch here in the future

This is definitely an improvement to be considered for wasm tests. Let's track that on a project issue.

Reply:
Sounds good https://github.com/stacks-network/clarity-wasm/issues/438